glade_forms/test/glade_input_test.dart

```diff
@@ -143,4 +143,48 @@ void main() {
       expect(input.value, equals(100));
     });
   });
+
+  group('Pure test', () {
+    test('ResetToPure', () {
+      final input = GladeInput.intInput(
+        value: 100,
```

Comment:
for the sake of the test, put explicitly initial value to some other value to check it?

Reply:
Done